ksql-version-metrics-client/src/main/java/io/confluent/ksql/version/metrics/KsqlVersionCheckerAgent.java

```diff
@@ -23,23 +23,27 @@
 
 import io.confluent.ksql.version.metrics.collector.KsqlModuleType;
 
-public class KsqlVersionCheckerAgent {
+public class KsqlVersionCheckerAgent implements VersionCheckerAgent{
+
+  private Thread versionCheckerThread;
+
+  protected KsqlVersionChecker ksqlVersionChecker;
```

Comment:
Looks like it can be `private`

Reply:
originally had it protected for some tests but then didn't it. Fixing it.